web/src/compose_pm_pill.ts

```diff
@@ -69,6 +69,17 @@ export function set_from_emails(value: string): void {
     widget.appendValue(value);
 }
 
+export function set_from_user_ids(value: number[]): void {
+    clear();
+    for (const user_id of value) {
+        const person = people.get_by_user_id(user_id);
+        user_pill.append_person({
+            pill_widget: widget,
+            person,
+        });
+    }
+}
```

Comment:
Should this be using `execute_oncreate_callback` or other techniques to avoid doing work for each user in the loop?

Reply:
I feel like there's a tradeoff here around simpler code (easier to understand and know that it works) vs. optimizing the work being done. And since we're almost always adding very few users here, I would lean towards keeping it as it is. How do you feel about that?

Reply:
Seems fine to me as long as it does not produce visual glitches of the users appearing one at a time.

Reply:
I just tried opening the composebox for a DM with over 10 participants and didn't see any visual glitch of that kind, so seems fine.